.github/workflows/migrate.yml

```
@@ -0,0 +1,66 @@
+            exit 1
+          fi
+
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Dependencies
+        run: |
+          npm install -g pnpm@9.15.3 && pnpm install
+
+      - name: Create .env file
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1046:error:1:1: Couldn't find 'fi' for this 'if'

(shellcheck)